libmamba/src/core/pool.cpp

```diff
@@ -160,6 +160,25 @@ namespace mamba
             return false;
         }
 
+        bool
+        subdir_match(std::string candidate_url, std::string needle_spec)
+        {
```

Comment:
Would be nice to have an example for `candidate_url` and  `needle_spec` in a comment here.